01-data-structures/02-stacks-and-queues/mystack/mystack_spec.rb

```diff
@@ -6,80 +6,99 @@
   let(:stack) { MyStack.new }
 
   describe "#push" do
-    it "pushes an item on top of the stack" do
-      expect(stack.stack_length).to eq 0
-      expect(stack.top).to eq nil
+    it "pushes a single item and checks the top of the stack stack height" do
       stack.push("Rob")
       expect(stack.top).to eq "Rob"
-      expect(stack.stack_length).to eq 1
     end
 
-    it "pushes an item on top of the stack" do
-      expect(stack.top).to eq nil
-      expect(stack.stack_length).to eq 0
+    it "pushes 5 items and checks the top of the stack and stack height" do
```

Comment:
each of these is a separate case. You are double testing a single push. 5 is an arbitrary number.